tests/integration/tc_71_8.c

```diff
@@ -9,15 +9,13 @@
 
 int main() {
     // Setup
-    start_test("Hotplug ForeignTestDevice", "", "", NO_REGEX);
+    start_test("Hotplug ForeignTestDevice", "", NO_REGEX);
 
     // Connect a ForeignTestDevice
     check_device_not_connected(UID);
     connect_virtual_device("ForeignTestDevice", UID);
     sleep(2);
     check_device_not_connected(UID);
 
-    // Clean up
-    end_test();
```

Comment:
this might also be old but why are the `end_test()` calls deleted in all of the integration tests?

Reply:
Yep this was implemented as a misc change in #185!